### ISSUE TITLE
Limitted chunk size when sending

### ISSUE THREAD

**Issue by qbolsee** [mraardvark/pyupdi#55]
Hi,

When using pyupdi to flash a ATtiny1614 using a SAMD11C, I experienced some very specific bug forcing me to send data in chunks smaller than 64 bytes. I documented the issue here:
[http://fabacademy.org/2020/labs/ulb/students/quentin-bolsee/projects/d11c_ftdi-updi/#arduino-core](http://fabacademy.org/2020/labs/ulb/students/quentin-bolsee/projects/d11c_ftdi-updi/#arduino-core)

I propose the following patch to handle this kind of issue:
[chunk_patch.txt](https://github.com/mraardvark/pyupdi/files/4604529/chunk_patch.txt)

Basically, the `send()` function in `physical.py` is modified to slice up data in chunks of a given max size. I think it's reasonable to use chunks when waiting for the echo, otherwise you can easily fill up some buffers on the USB serial, leading to strange behaviors.
